src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
+export enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
 
+export interface Figure {
+  shape: Shape;
+  color: 'red' | 'green' | 'blue';
```

Comment:
create a separate type for Color

```suggestion
  color: Color;
```